src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,126 @@
+import React,{useState,useEffect} from 'react'
+import { Box, Grid, Typography } from '@mui/material'
+import Image from 'next/image';
+import { navItem,getIntouch} from '@/constants';
+
+
+const Footer = () => {
+    const [width, setWidth] = useState(true);
```

Comment:
Why a flag is been used here, responsive styles can be given in mui components directly
please check the responsive-demo [here](https://mui.com/system/getting-started/usage/#responsive-demo)

Reply:
@abisheksridhar , i've tried it, it's working for me attaching the syntax below
`fontSize: { xs: '150px', md: '300px' },`